client/app/projects/profile/profile.controller.js

```diff
@@ -200,6 +200,29 @@ angular.module('observatory3App')
             });
         }
     };
+//tech bubble code
+	$scope.isMentor = Auth.isMentor;
```

Comment:
why do we only want mentors to add remove tech bubbles? Shouldn't anyone on the project be able to?